components/ipfs/ipfs_utils.cc

```diff
@@ -18,7 +18,6 @@
 #include "brave/components/ipfs/pref_names.h"
 #include "components/base32/base32.h"
 #include "components/prefs/pref_service.h"
-#include "extensions/common/url_pattern.h"
```

Comment:
ipfs doesn't have a dep for including this anywhere right? If so it'd need to be removed as well

Reply:
sure, thanks